install_config/cfme/role_variables.adoc

```diff
@@ -0,0 +1,267 @@
+[[install-config-cfme-role-variables]]
+= Configuring Role Variables
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+ifdef::openshift-enterprise[]
+:cloudforms: Red Hat CloudForms
+endif::[]
+ifdef::openshift-origin[]
+:cloudforms: ManageIQ
+endif::[]
+
+toc::[]
+
+== Overview
+
+The following sections describe role variables that may be used in your Ansible
+inventory files to control the behavior of the {cloudforms} installer.
+
+[IMPORTANT]
+====
+Not all parameters are present in _both_ template versions (containerized or
+external database). For example, while the podified database template has a
+`POSTGRESQL_MEM_REQ` parameter, no such parameter is present in the external db
+template, as there is no need for this information due to there being no
+databases that require pods.
+
+Therefore, be very careful if you are overriding template parameters. Including
+parameters not defined in a template will cause errors. If you do receive an
+error during the `Ensure the CFME App is created` task, run the
+xref:uninstalling.adoc#install-config-cfme-uninstalling[uninstall scripts] first before running the installer again.
+====
+
+== General Variables
+
+[options="header",cols="1,1,1,4"]
+|===
+|Variable |Required |Default |Description
+
+|`openshift_management_install_management` 
+|No
+|`false`
+|Boolean, set to `true` to install the application.
+
+ifdef::openshift-enterprise[]
+|`openshift_management_install_beta`
+|No
+|`false`
+|By setting this value to `true`, you acknowledge that the {cloudforms} 4.6
+software is currently in beta, and support may be limited. This is required to
+begin the installation.
+endif::[]
+
+|`openshift_management_project`
+|No 
+|`openshift-management` 
+|Namespace (project) for the {cloudforms} installation.
+
+|`openshift_management_project_description`
+|No 
+|`CloudForms Management Engine`
+|Namespace (project) description.
+
+|`openshift_management_username`
+|No 
+|`admin` 
+|Default management user name. Changing this value does not change the username; only change this value if you have changed the name already and are running integration scripts (such as the script to xref:cloud_provider.adoc#install-config-cfme-cloud-provider[add cloud providers]).
+
+|`openshift_management_password`
+|No
+|`smartvm`
+|Default management password. Changing this value does not change the password; only change this value if you have changed the password already and are running integration scripts (such as the script to xref:cloud_provider.adoc#install-config-cfme-cloud-provider[add cloud providers]).
+|===
+
+== Database Variables
+
+[options="header"]
+|===
+|Variable |Required |Default |Description
+|`openshift_management_app_template`
```

Comment:
openshift_management_app_template should also be a "General Variable", so it's better to list openshift_management_app_template in the above table. 
And we should have 4 options for this variable.
Pls check   "PRODUCT CHOICE : openshift_management_app_template"  in 
https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_management#role-variables

Reply:
I will move it to General Variables. There already are 4 total options, it's just that 2 of them show up for OCP (for CFME), and the other 2 for Origin (MIQ).